tailwind.config.js

```diff
@@ -13,7 +13,26 @@ module.exports = {
             ],
             serif: ['Noto Serif TC', '-apple-system', 'serif'],
         },
-        extend: {},
+        extend: {
+            width: {
+                '272px': '272px',
+                '706px': '706px',
+                '1040px': '1040px',
+            },
+            padding: {
+                '24px': '24px',
+                '32px': '32px',
+                '168px': '168px',
+            },
```

Comment:
If we're already using [arbitrary value support](https://v2.tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support), do we still need to specify the values in the config?
It looks redundant to me.